pkg/tsdb/loki/testdata_metric_dataplane/empty.golden.jsonc

```diff
@@ -0,0 +1,5 @@
+//  🌟 This was machine generated.  Do not edit. 🌟
+//  🌟 This was machine generated.  Do not edit. 🌟
+{
+  "status": 200
+}
```

Comment:
If "empty" means "no data" in Grafana terms here, it can be returned as a single frame with no fields. This way metadata can still be passed. In particular, the executed query string (if using macros, can help to see what happened to result in no data if unexpected). https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/contract_docs/contract.md#no-data-and-empty